### PR TITLE
docs: Added an info about custom nameservers for the kubelet wrapper

### PR DESCRIPTION
**Pull request by kayrus**
@joshix @robszumski

### CODE REVIEW

**Comment by robszumski** [comment]
What do you think of making this section more generic, like "Customizing rkt options"? I foresee a lot other small requests popping up.

<hr>

### Customizing rkt options

Passing customized options or flags to rkt can be accomplished with the `RKT_OPTS` environment variable. Referencing it in a unit file is straightforward:

```ini
[Service]
Environment=RKT_OPTS="--dns 1.2.3.4"
Environment=KUBELET_VERSION=v1.1.8_coreos.0
ExecStart=/usr/lib/coreos/kubelet-wrapper \
  --api-servers=http://127.0.0.1:8080 \
  --config=/etc/kubernetes/manifests
```

#### Example: Use custom DNS servers

By default, the hyperkube container uses Google DNS IP (8.8.8.8). You can modify this behavior in rkt 1.2+:

```ini
Environment=RKT_OPTS="--dns 1.2.3.4"
```

For rkt 1.1 and older, you can use this longer syntax (bug: [#2141][#2141]):

```ini
Environment=RKT_OPTS="--volume=resolv,kind=host,source=/etc/resolv.conf --mount volume=resolv,target=/etc/resolv.conf"
```

**Comment by kayrus** [comment]
@robszumski milestone has been changed to 1.4.0 (https://github.com/coreos/rkt/issues/2141). Will just remove info about the `--dns 1.2.3.4.`

**Comment by aaronlevy** [comment]
This LGTM. @robszumski were we waiting on anything to merge this?

**Comment by robszumski** [comment]
@aaronlevy this PR doesn't contain the example, which seems like the primary use-case, no?

**Comment by robszumski** [comment]
Note that we can remove the 1.2 vs 1.1 stuff since these docs are versioned. We can just cover the simple 1.2 use-case :+1:

**Comment by kayrus** [comment]
@robszumski it was removed.

**Comment by joshix** [comment]
This seems good to me. @robszumski Is there another example you wanted added, or has that comment been resolved?

File: Documentation/kubelet-wrapper.md [hunk 1]
```diff
@@ -27,21 +27,37 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
 
 In the example above we set the `KUBELET_VERSION` and the kubelet-wrapper script takes care of running the correct container image with our desired API server address and manifest location.
 
+### Customizing rkt Options
+
+Passing customized options or flags to rkt can be accomplished with the RKT_OPTS environment variable. Referencing it in a unit file is straightforward. Example below mounts host's `/etc/resolv.conf` file directly into the container:
+
+```ini
+[Service]
+Environment=RKT_OPTS="--volume=resolv,kind=host,source=/etc/resolv.conf --mount volume=resolv,target=/etc/resolv.conf"
```
Comment (robszumski) [inline Documentation/kubelet-wrapper.md]: Isn't this the 1.1.x example not the 1.2 one, which was simpler? We've moved to 1.2 for master.

File: Documentation/kubelet-wrapper.md [hunk 2]
```diff
@@ -27,21 +27,37 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
...
+Environment=KUBELET_VERSION=v1.1.8_coreos.0
```
Comment (robszumski) [inline Documentation/kubelet-wrapper.md]: 1.2 not 1.1.8

File: Documentation/kubelet-wrapper.md (hunk 1 above)
Comment (kayrus) [inline Documentation/kubelet-wrapper.md]: `--dns` will be fixed in 1.4. Please check the issue: https://github.com/coreos/rkt/issues/2141
And initial commit contained info about `please use rkt greater than 1.2 (>1.2, not >=1.2)`. Them milestone was changed from 1.3 to 1.4 and I decided to completely remove `--dns`.

File: Documentation/kubelet-wrapper.md (hunk 2 above)
Comment (kayrus) [inline Documentation/kubelet-wrapper.md]: Fixed.

**Comment by joshix** [comment]
LGTM with @kayrus's note about delay of --dns option for stage1. @robszumski: I say this should be merged, but don't want to be peremptory, so PTAL then PTBF.

INITIALISMS FTW!

**Comment by robszumski** [comment]
LGTM, thanks for keeping it on track @kayrus

**Pull request merged**